app/serializers/disability_claim_detail_serializer.rb

```diff
@@ -9,4 +9,58 @@ def contention_list
   def va_representative
     object.data['poa']
   end
+
+  def events_timeline
+    events = [
+      create_event_from_string_date(:filed, 'date'),
+      create_event_from_string_date(:completed, 'claimCompleteDate')
+    ]
+
+    # Do the 8 phases
+    (1..8).each do |n|
+      events << create_event_from_string_date(
+        "phase#{n}", 'claimPhaseDates', "phase#{n}CompleteDate"
```

Comment:
There was some discussion of narrowing the number of phases from 8 to 3 or 5. If the UI is not showing all 8 phases, this event will be confusing for the phases that aren't shown.